src/main/java/uk/gov/justice/hmpps/prison/api/model/v1/AccountBalance.java

```diff
@@ -1,15 +1,16 @@
 package uk.gov.justice.hmpps.prison.api.model.v1;
 
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.Hidden;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
 
-@ApiModel(description = "Account Balance")
+@Schema(description = "Account Balance")
```

Comment:
the `v1` model files use stored procedures and map through intermediate objects so performance is unaffected